roles/tm-deepsec-aws-setup/vars/main.yml

```diff
@@ -0,0 +1,34 @@
+# vars file for tm_silent_setup
+---
+db:
+  db_user: dsm
+  db_name: ansible
+  db_password: ansible
+
+pre_package:
+  - bash
+  - openssl
+  - python-devel
+  - gcc
+  - python-pip
+  - postgresql-devel
+
+pip_package:
+  - psycopg2
+
+dir_permission:
+  - /usr/lib64/python2.7/site-packages/psycopg2
+  - /usr/lib64/python2.7/site-packages/psycopg2-2.8.6-py2.7.egg-info
+
+uninstall_postgres:
+  - python-devel
+  - postgresql
+  - postgresql-server
+
+remove_dir:
+  - /var/lib/pgsql
+
+postgres_rpm:
```

Comment:
May I suggest to use the official name in the parameter name: `postgresql_rpm`

Reply:
+1